### PR TITLE
Remove leftover data files

### LINKED ISSUE #50: Remove leftover data directory
**Is this a BUG REPORT or FEATURE REQUEST?**:
bug

**What happened**:
After the migration the data directory is not necessary in exampleSite anymore.
 
**What you expected to happen**:
No data directory in exampleSite.

### PR DESCRIPTION
**Pull request by lauragift21**
**What this PR does / why we need it**: 
Removes all data files after migrating to content structure.

**Which issue this PR fixes** *(optional - uncomment and add issue)*:
fixes #50

### CODE REVIEW

File: exampleSite/config.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ googleAnalytics = ""
 [params]
   name = "Okkur Labs"
   description = "Open Source Theme for your next project"
-  attribution = "true"
+  attribution = "false"
```
Comment (stp-ip) [inline exampleSite/config.toml]: Do we want to make attribution opt-in? Reasoning behind this change?

**Comment by stp-ip** [comment]
Will merge as the other PR was also merged, but for each new feature a new branch with a fresh base is easier to review.

File: exampleSite/config.toml (hunk 1 above)
Comment (lauragift21) [reply exampleSite/config.toml]: I could switch that back to true but add a comment to either enable true or false.

Comment (stp-ip) [reply exampleSite/config.toml]: Think it might be best to discuss that. Let's leave it as opt-in for now. Discussion is at #53

**Pull request merged**